web-ui/src/components/guild-results/GuildSummaryCard.jsx

```diff
@@ -275,6 +279,18 @@ const GuildSummaryCard = ({ guild, index, isOpen, onGuildSelect }) => {
               type: UPDATE_GUILDS,
               payload: copy
             });
+            const resGetGuildLeaders = await getGuildLeaders(editedGuild.id, csrf);
+            const guildLeaders = resGetGuildLeaders.payload?.data && !resGetGuildLeaders.error
+                ? resGetGuildLeaders.payload.data
+                : null;
+            if (guildLeaders && oldGuildLeaders) {
+              // Filter out the new leaders that were not in the old leaders
+              const newLeaders = guildLeaders.filter(
+                  newLeader => !oldGuildLeaders.some(oldLeader => oldLeader.id === newLeader.id)
+              );
+
+              newLeaders.length > 0 && emailGuildLeaders(newLeaders, guild, csrf).then();
```

Comment:
Hanging `then()` here and two more in `AdminMemberCard`. Those can safely be removed.

Reply:
Fixed